aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -421,27 +486,103 @@ module aptos_framework::aptos_governance {
         metadata
     }
 
+    fun add_is_multi_step_proposal_flag(metadata: &mut SimpleMap<String, vector<u8>>, is_multi_step_proposal_value: vector<u8>) {
+        assert!(is_multi_step_proposal_value == MULTI_STEP_PROPOSAL || is_multi_step_proposal_value == SINGLE_STEP_PROPOSAL, error::invalid_argument(EPROPOSAL_METADATA_INVALID_KEY));
+        simple_map::add(metadata, utf8(IS_MULTI_STEP_PROPOSAL_KEY), is_multi_step_proposal_value);
```

Comment:
Why not just a simple boolean instead of vec<u8> for is_multi_step_proposal_value?

Reply:
it's bc the metadata map is of type `<String, vector<u8>>`